### ISSUE TITLE
[feature] add 'docker-composer' plugin bundler

### ISSUE THREAD

**Issue by itsNikolay** [robbyrussell/oh-my-zsh#3557]
https://github.com/docker/fig
Docker team took the `fig` tool to it's support, and renamed to `docker-composer`
could somebody add aliases for it, it's very usefull tool which will replace Vagrant in further.
many thanks

**Comment by robbyrussell** [comment]
@itsNikolay Want to submit a PR for this? You can look at existing plugins for examples

**Comment by itsNikolay** [comment]
@robbyrussell sure, I will try it

**Comment by wernight** [comment]
bashcompinit gives some errors with the Bash autocomplete provided:

````
_docker-compose:20: command not found: _get_comp_words_by_ref
_docker-compose:27: parse error: condition expected: 1
_docker-compose:declare:48: not valid in this context: _docker-compose_docker-compose
...
````

Suggesting to use argparse in [docker-compose](https://github.com/docker/compose/issues/1150)

**Comment by paulodiovani** [comment]
Could this one be used? https://github.com/sdurrheimer/docker-compose-zsh-completion.git

I added in `plugins` and it's working fine.

**Comment by wernight** [comment]
Using another projet's auto-complete will lead to out of sync auto-complete.

It should be supported natively.

**Comment by mcornella** [comment]
Solved in #4099, please close this issue.
